src/main/resources/application.yml

```diff
@@ -87,8 +87,17 @@ management:
       cache:
         time-to-live: 2000ms
 
+
+services:
+  prisons-api:
+    base-url: http://localhost:9094
```

Comment:
Which of these is correct here? If one is needed for a different environment then should we move it in a different `application-x.yaml` file?

Reply:
This is something which am still trying to figure out. We have `application.yaml`, `application-local.yaml`,`application-test.yaml` 

Trying to work out which yaml takes precedence

Reply:
The tests read from application-test.yaml without these `services:` block the tests don't start up.
Application.yaml takes precedence. 

Does this answer your question?